x/auth/ante/sigverify_internal_test.go

```diff
@@ -20,7 +20,7 @@ type mockAccount struct {
 	ante.AccountKeeper
 }
 
-func (*mockAccount) Environment() appmodule.Environment {
+func (*mockAccount) GetEnvironment() appmodule.Environment {
```

Comment:
This is a method on the keeper, not a service. As we use an interface instead of the concrete type for ante handler, we need to obtain environment. It was just called `Environement()` before but now it clashes with the embedded environment on the auth struct.